src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -261,19 +245,17 @@ class OrganizationIntegrations extends AsyncComponent<
         />
       );
     }
-
-    return (
-      <SentryAppInstallationDetail
-        key={`sentry-app-row-${app.slug}`}
-        data-test-id="integration-row"
-        api={this.api}
-        organization={organization}
-        install={this.getAppInstall(app)}
-        onAppUninstall={() => this.handleRemoveAppInstallation(app)}
-        onAppInstall={this.handleAppInstallation}
-        app={app}
-      />
-    );
+    if (app.status === 'published') {
+      return (
+        <IntegrationDirectoryAppInstallationDetail
```

Comment:
@NisanthanNanthakumar why can't we just use `IntegrationDirectoryApplicationRow` here and pass in `isOnIntegrationPage`?